src/components/CheckboxInput/CheckboxInput.tsx

```diff
@@ -77,22 +51,14 @@ export const CheckboxInput = React.forwardRef<
         };
 
   return (
-    <span
-      className={clsx(
-        styles['input__wrapper'],
-        disabled && styles['input__wrapper--disabled'],
-      )}
-    >
-      <input
-        className={clsx(className, styles['checkbox__input'])}
-        disabled={disabled}
-        ref={ref}
-        type="checkbox"
-        {...checkedProps}
-        {...other}
-      />
-      <CheckboxSvg checked={checked} disabled={disabled} />
-    </span>
+    <input
+      className={clsx(className, styles['checkbox__input'])}
```

Comment:
Not related to the changes, but the class name should be `styles.input`.

The BEM-ing isn't doing anything for us, except making us type more.

It's like we took the `checkedProps` JS variable in this file and instead named it `checkbox__input__checkedProps`.

`</rant>`